plane/source/docs/airspeed-parameters-setup.rst

```diff
@@ -0,0 +1,40 @@
+.. _airspeed-parameters-setup:
+
+=========================
+Airspeed Parameters Setup
+=========================
+
+.. note: ArduPlane does not require an airspeed sensor. However, some airspeeed related parameters are used even if no airspeed sensor is present or being used, notably for the scaling of tuning parameters with speed.
+
+
+Speed Scaling
+=============
+
+Since the effect of flying surface control deflection increases and decreases with airspeed, the stabilization gains in the PID loops are automatically scaled with airspeed by ArduPilot to prevent oscillations at higher airspeeds and preserve stabilization effectiveness at low airspeeds, even if no airspeed sensor is used.
+
+This is accomplished by Ardupilot estimating airspeed from other sensor inputs like GPS velocity and position changes and IMU accelerations. This estimate is also used in case an airspeed sensor is used and becomes unhealthy. In systems without an airspeed sensor this estimate is used for speed scaling of the PID loops.
+
+So, even if an airspeed sensor is not present, some airspeed parameters are used. Normally, the default values are acceptable, but very fast or slow vehicles may need them adjusted for best performance.
+
+Parameters
+==========
+
+Key parameters are:
+
+- :ref:`ARSPD_FBW_MAX<ARSPD_FBW_MAX>` : This is the fastest normal flying speed for the vehicle. Normally, 2 times the nominal cruising speed is a good target.
+- :ref:`ARSPD_FBW_MIN<ARSPD_FBW_MIN>` : This is the slowest flying speed for the vehicle. It should be set at least 25% above the stall speed of the vehicle.
+- :ref:`SCALING_SPEED<SCALING_SPEED>` : This is the center of the speed scaling range and should be set close to the normal cruising speed of the vehicle. PID tuning (see :ref:`tuning-quickstart`) should be done at this flying speed. The loop gains are scaled above and below this value with a multiplier determined base on the above min and max flying speeds.
```

Comment:
we need a warning that changing the SCALING_SPEED will change the tuning of the aircraft by a large amount. It is recommended not to change SCALING_SPEED after your aircraft is tuned.

Reply:
added